src/m_pd.c

```diff
@@ -129,21 +176,19 @@ void pd_bind(t_pd *x, t_symbol *s)
         if (*s->s_thing == bindlist_class)
         {
             t_bindlist *b = (t_bindlist *)s->s_thing;
-            t_bindelem *e = (t_bindelem *)getbytes(sizeof(t_bindelem));
-            e->e_next = b->b_list;
-            e->e_who = x;
-            b->b_list = e;
+            int oldsize = b->b_n++;
+            b->b_vec = (t_pd **)resizebytes(b->b_vec, oldsize * sizeof(t_pd *),
+                                   b->b_n * sizeof(t_pd *));
+            memmove(b->b_vec + 1, b->b_vec, oldsize * sizeof(t_pd *));
```

Comment:
one of the CI-builds rightfully fails, as `memmove` and `memcpy` are not declared tue to a missing `#include <string.h>`

Reply:
d'oh. thanks!